src/Text/Tags.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Text.Tags where
+
+import Data.Attoparsec.Text
+       (Parser, parseOnly, letter, space, char, sepBy, many1)
+import Control.Applicative ((<|>))
+import Prelude hiding (take, takeWhile)
+import Types
+import Data.Text (Text, pack)
+
+
+tagOp :: Parser TagOp
+tagOp =
+  (char '+' >> AddTag <$> (pack <$> many1 letter))
+  <|> (char '-' >> RemoveTag <$> (pack <$> many1 letter))
+  <|> (char '=' >> pure ResetTags)
+
+allTagOps :: Parser [TagOp]
+allTagOps = tagOp `sepBy` space
```

Comment:
This will currently return always a `Right []`, but perhaps for the future it should fail for `foo bar baz`, since the operators are missing. If you have a recommendation how to do it better I'm all ears, otherwise I'll put it in a separate backlog item since any failures are currently not exposed to the UI anyways.

Reply:
True... hm... weird I thought I had tried this ...